.github/templates/issue_template.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Issue Report
+about: Create a bug report to help us resolve it!
+title: "[ISSUE]"
+labels: ''
+assignees: '@-'
+
+---
+
+<h4>Summary</h4>
+Please provide a concise summary of the bug/issue.
+
+
+<h4>Expected Behavior</h4>
+Provide a concise summary of what was expected to happen.
+
+
+<h4>Actual Behavior</h4>
+Provide a concise summary of what actually happened.
+
+
+<h4>To Reproduce</h4>
```

Comment:
@sglibova - Based on what I'm reading in this template, the steps to reproduce sound like we're expecting browser issues or UI problems. For a client library I'd expect slightly different examples.